apps/wallet/src/ui/app/hooks/useCreateAccountMutation.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useMutation } from '@tanstack/react-query';
+import { toast } from 'react-hot-toast';
+import { useBackgroundClient } from './useBackgroundClient';
+
+export function useCreateAccountsMutation() {
+	const backgroundService = useBackgroundClient();
+	return useMutation({
+		mutationKey: ['create accounts'],
+		mutationFn: (...params: Parameters<typeof backgroundService.createAccounts>) =>
+			backgroundService.createAccounts(...params),
+		onError: (error) => {
+			toast.error((error as Error)?.message || 'Failed to create account. (Unknown error)');
+		},
+		onSuccess: (result) => {
```

Comment:
no need to change this + I don't think it really matters here, but fun fact I learned the other day - they say you should put non-UI actions in the useMutation `onSuccess` / `onError` options and put UI actions in the mutate function's `onSuccess` / `onError` 

https://tkdodo.eu/blog/mastering-mutations-in-react-query#some-callbacks-might-not-fire